src/covid19model/visualization/output.py

```diff
@@ -0,0 +1,179 @@
+                        "blue" : "#0072B2", "red" : "#D55E00",
+                        "pink" : "#CC79A7", "black" : "#000000"}
+
+# covid 19 specific parameters
+plt.rcParams.update({
+    "axes.prop_cycle": plt.cycler('color',
+                                  list(colorscale_okabe_ito.values())),
+    "font.size": 15,
+    "lines.linewidth" : 3,
+    "axes.spines.top": False,
+    "axes.spines.right": False,
+    "ytick.major.left": True,
+    "axes.grid": True
+})
+
+
+def _apply_tick_locator(ax):
+    """support function to apply default ticklocator settings"""
+    ax.xaxis.set_major_locator(plt.MaxNLocator(5))
+    ax.yaxis.set_major_locator(plt.MaxNLocator(4))
+    return ax
+
+
+def population_status(data, filename=None, *, ax=None, **kwargs):
+    """Plot evolution of the population as function of time
+
+    The function is currently tailor-made for the SeirsAge model,
+    (but could be more generalized in the future). As such, it requires
+    the following variable names from the SeirsAge model:
+    S, E, R, C, Cmirec, Cicurec, I, MI, ICU, A, M.
+
+    # TODO - should work with MC-based version as well, using percentiles
+
+    Parameters
+    ----------
+    data : xarray DataSet
+        Model output
+    ax : matplotlib.axes.Axes, optional
+        If provided, will use the axis to add the lines.
+    **kwargs :
+        Keyword arguments are passed to xarray/matplotlib line function
+
+    Returns
+    -------
+    lines : list of 4 Line Artists
+        Each of the lines can be furhter customized by the user
+    """
+    # check the required variable names to let function work
+    required_var_names = ["S", "E", "R",
+                          "C", "Cmirec", "Cicurec",
+                          "I","Mi", "ICU", "A", "M"]
+    if not (set(required_var_names)).issubset(set(data.variables.keys())):
+        raise Exception(f"population_status plot function"
+                        f"requires the variables {required_var_names}")
+
+    # create the 'combined variables'
+    # TODO abstract away 'observed' or 'states of interest' to higher level
+    data["C_total"] = data["C"] + data["Cmirec"] + data["Cicurec"]
+    data["I_total"] = (data["Mi"] + data["ICU"] + data["I"] +
+                       data["A"] + data["M"] + data["C_total"])
+
+    # stratified models are summarized over stratification dimension
+    if "stratification" in data.dims:
+        data = data.sum(dim="stratification")
+
+    # check if ax object is provided by user
+    if ax is None:
+        fig, ax = plt.subplots()
+
+    # create plot using xarray interface
+    data2plot = data[["S", "E", "I_total", "R"]].to_array(dim="states")
+    lines = data2plot.plot.line(x='time', hue="states", ax=ax, **kwargs)
+    ax.set_xlabel('days')
+    ax.set_ylabel('number of patients')
+
+    # use custom defined colors
+    colors = ["black", "orange", "red", "green"]
+    for color, line in zip(colors, lines):
+        line.set_color(colorscale_okabe_ito[color])
```

Comment:
this should not be needed if the `colorscale_okabe_ito` was using this order, I think?